src/incubator/Dialog/__tests__/index.new.spec.tsx

```diff
@@ -98,4 +99,11 @@ describe('Incubator.Dialog sanity checks', () => {
     expect(await dialogDriver.getModal().isVisible()).toBeFalsy();
     expect(onDismiss).toHaveBeenCalledTimes(1);
   });
+
+  it('Modal exists even when not visible (WOAUILIB-3606)', async () => {
```

Comment:
Same here for the async
```suggestion
  it('Modal exists even when not visible (WOAUILIB-3606)', () => {
```